ansible/roles/vm_set/library/vlan_port.py

```diff
@@ -150,12 +144,13 @@ def main():
 
     fp_port_templ = external_port + ".%s"
     if is_multi_duts:
-        fp_ports = []
+        fp_ports = {}
         for dut_vlans in vlan_ids:
             dut_vlans.sort()
             fp_ports.append([fp_port_templ % vid for vid in dut_vlans])
```

Comment:
Should we ensure the same behavior regarding multiple-DUTs' testbed?

Reply:
I am not sure if is_multi_duts is still being used. All calls to vlan_port from ansible/roles/vm_set do not pass is_multi_duts and are thus false.  But, I am not aware of how this is being used by SpyTest. If they don't use it, then we can remove support multi_dut from vlan_port as well

Reply:
Looks like the `vlan_port` will never be called with the nested `vlan_ids` since it is called with a loop.

Reply:
should we remove is_multi_duts support from vlan_port then.